static/app/routes.tsx

```diff
@@ -1922,13 +1934,21 @@ function buildRoutes() {
       <IndexRoute
         component={make(() => import('sentry/views/feedback/feedbackListPage'))}
       />
+      <Route
+        path="trace/:traceSlug/"
+        component={make(() => import('sentry/views/performance/traceDetails'))}
+      />
     </Route>
   );
 
   const issueListRoutes = (
     <Route path="/issues" component={errorHandler(IssueListContainer)} withOrgPath>
       <IndexRoute component={errorHandler(OverviewWrapper)} />
       <Route path="searches/:searchId/" component={errorHandler(OverviewWrapper)} />
+      <Route
+        path="trace/:traceSlug/"
+        component={make(() => import('sentry/views/performance/traceDetails'))}
+      />
```

Comment:
Instead of duplicating this everywhere, I think a shared const would be better

Reply:
Yep I agree, I'll make that change